src/main/java/io/appium/java_client/AppiumBy.java

```diff
@@ -257,4 +323,56 @@ protected ByIosNsPredicate(String locatorString) {
             super("-ios predicate string", locatorString, "iOSNsPredicate");
         }
     }
+
+    public static class FlutterBy extends AppiumBy {
+
+        protected FlutterBy(String selector, String locatorString, String locatorName) {
+            super(selector, locatorString, locatorName);
+        }
+
+        /**
+         * Parses an {@link AppiumBy.FlutterBy} object into a Map of parameters
+         * to be executed by flutter integration driver.
+         *
+         * @return A {@link Map} containing the parsed parameters
+         */
+        public Map<String, Object> toJson() {
```

Comment:
Lets add this method with the driver, so it would be better visible where and how it is used